api/camunda/camunda-openapi.yaml

```diff
@@ -0,0 +1,549 @@
+openapi: "3.0.3"
+info:
+  title: Camunda 8 REST API
+  version: "0.1"
+  description: API for communicating with the Camunda 8 cluster.
+  license:
+    name: Camunda License Version 1.0
+    url: https://github.com/camunda/camunda/blob/main/licenses/CAMUNDA-LICENSE-1.0.txt
+externalDocs:
+  description: Find out more
+  url: https://docs.camunda.io/docs/apis-tools/camunda-api-rest/overview/
+
+servers:
+  - url: "{schema}://{host}:{port}/v2"
+    variables:
+      host:
+        default: localhost
+        description: The hostname of the C8 REST Gateway.
+      port:
+        default: "8080"
+        description: The port of the C8 REST API server.
+      schema:
+        default: http
+        description: The schema of the C8 REST API server.
+
+paths:
+  /topology:
+    get:
+      tags:
+        - Cluster
+      summary: Get cluster topology
+      description: Obtains the current topology of the cluster the gateway is part of.
+      responses:
+        "200":
+          $ref: "#/components/responses/TopologyResponse"
+  /jobs/activation:
+    post:
+      tags:
+        - Job
+      summary: Activate jobs
+      description: |
+        Iterate through all known partitions and activate up to the requested maximum and
+        stream them back to the client as they are activated.
```

Comment:
I'd like to see the description include "jobs." The way it reads now I'm not sure if the "requested maximum" is partitions, jobs, or something else.

I also would avoid using "stream" as it may confuse users that this is a streaming API (vs. a REST API).

Reply:
Good catch, @akeller, I'll definitely adjust this 👍